swagger.json

```diff
@@ -0,0 +1,3445 @@
+{
```

Comment:
I don't think we want this file in the repo.

Reply:
Removed.  Oops